### ISSUE TITLE
Source request (Multi): Niadd

### ISSUE THREAD

**Issue by ghost** [keiyoushi/tachiyomi-extensions-source#3886]
### Source name

Niadd

### Source link

https://br.niadd.com

### Source language

Português (Brasil)

### Other details

18+/NFSW = yes

### Acknowledgements

- [X] I have checked that the extension does not already exist by searching the [extension listing](https://keiyoushi.github.io/extensions/) and verified it does not appear there.
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a meaningful title with the source name.
- [X] I will fill out all of the requested information in this form.

**Comment by vetleledaal** [comment]
Site supports English, Español, lingua italiana, Русский язык, Português, Deutsch, and Français. There's a language selector top right.

**Comment by ghost** [comment]
> Site supports English, Español, lingua italiana, Русский язык, Português, Deutsch, and Français. There's a language selector top right.

Não tinha visto, desculpa aí

**Comment by choppeh** [comment]
It looks like a mirror from [NineManga](https://keiyoushi.github.io/extensions/#all.ninemanga)

**Comment by JhulShock** [comment]
please add support for this source

**Comment by Zev-Kaapor** [comment]
Oi! 😅 Sobre o Niadd BR que você mencionou no issue…
Eu dei uma olhada e infelizmente não tem como implementar o suporte multi-idioma por enquanto. O site depende muito de JavaScript pra carregar o conteúdo, e a maneira que o Tachiyomi lê as páginas não consegue processar isso de forma confiável.
Tentamos contornar de algumas formas, mas acabamos esbarrando em problemas de consistência e estabilidade. Por enquanto, a extensão só vai funcionar no inglês, que é a versão mais estável do site.
Se no futuro o site mudar a forma de carregar o conteúdo, a gente pode tentar reavaliar. Até lá, é o que dá pra fazer 😅.
Para mais detalhes sobre a implementação e possíveis contribuições, veja o arquivo **Docs-NiaddNotes.md** na extensão.

---

Hi! 😅 About the Niadd BR you mentioned in the issue…
I took a look and unfortunately it’s not possible to implement multi-language support right now. The site relies heavily on JavaScript to load content, and the way Tachiyomi reads pages can’t reliably handle that.
We tried some workarounds, but ran into consistency and stability issues. For now, the extension will only work in English, which is the site’s most stable version.
If the site changes how content is loaded in the future, we can reconsider. Until then, that’s what’s possible 😅.
For more details about the implementation and possible contributions, check the **Docs-NiaddNotes.md** file in the extension.

**Comment by stevenyomi** [comment]
> The site relies heavily on JavaScript to load content, and the way Tachiyomi reads pages can’t reliably handle that

That's not true. You can send any kind of HTTP request from the extension and parse the response besides plain HTML. Come to our Discord if you have question.

**Comment by Zev-Kaapor** [comment]
> That's not true. You can send any kind of HTTP request from the extension and parse the response besides plain HTML. Come to our Discord if you have question.

Yes, that's correct — Tachiyomi extensions can indeed send any type of HTTP request, not just plain HTML. However, in the case of **Niadd**, the main issue isn’t just “using JavaScript,” but rather how the site itself structures and exposes its content.

During testing, we ran into multiple issues:

- Pages failing to return all images properly  
- Inconsistent responses depending on the selected language  
- Overall instability when trying to keep multi-language support  

Because of that, we decided to stick with the **English version only**, since it’s the most stable and reliable for the extension.

I’ve also added some extra notes in the file **`Docs-NiaddNotes.md`** inside the PR, in case anyone wants to investigate further or attempt an alternative solution later.

I’m also open to discussing this further on Discord, but I wanted to make sure the reasoning was documented here as well.

**Comment by Athrun0311** [comment]
Se puede abrir una solicitud para agregar la versión español de la, fuente  esnidd  https://es.niadd.com/category/manhwa.html

![Image](https://github.com/user-attachments/assets/67a35356-ca06-4024-9334-d4c0558673c2). Saludos
